### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jinpark**
Remove references to CI and auto deployment. 

Clarify on how to review PRs. 

Remove reference to releases.md.

Remove html from markdown files

### CODE REVIEW

**Comment by jinpark** [comment]
@RachellCalhoun @JiwonHwang @hassanabidpk take a look please

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-<h1>Light and Leadership Non-profit Project</h1>
+Light and Leadership Non-profit Project
```
Comment (jinpark) [inline README.md:1]: @RachellCalhoun So this page is a `.md` file which stands for markdown

Markdown is more or less a simplified version of html with its own rules.

Check it out at https://guides.github.com/features/mastering-markdown/

**Comment by hassanabidpk** [comment]
Reminder :1234:

**Comment by RachellCalhoun** [comment]
Yep. Had a look. Reminder?  Am I supposed to do something?

**Pull request merged**